### ISSUE TITLE
Non-specific object naming on grid objects

### ISSUE THREAD

**Issue by callistosp** [metrumresearchgroup/babylon#141]
Problem: currently `babylon` creates an object named `grid.sh` that is added to the grid. If multiple models are sent to the grid, when checking their status using `qstat` it is impossible to discern which model is which.

```
samc$ qstat
job-ID  prior   name       user         state submit/start at     queue                          slots ja-task-ID
-----------------------------------------------------------------------------------------------------------------
      2 0.55500 grid.sh    samc         r     03/03/2020 21:46:25 all.q@ip-x.y.z.t-     1
      3 0.00000 grid.sh    samc         qw    03/03/2020 21:46:28                                    1
```

This issue makes it difficult to selectively terminate models running on the grid. Ideally objects on the grid would be named identically to the model from which they were derived.

**Comment by dpastoor** [comment]
Agree, this is an oversight, I thought that is how it was actually. @shairozan we should update the template script to use model name as the default name

**Comment by shairozan** [comment]
Relatively simple change to make. Just use the `-N` switch for qsub, which poses it's own problem. If the model name begins with a digit, qsub will complain about it not being valid. The worst part about this is that qsub doesn't actually return an error exit code for this scenario. 

@dpastoor  should we check the model filename to see if it starts with a digit? And if so prepend a prefix? Or should we just look at always prepending a prefix?

**Comment by callistosp** [comment]
In our current system it appends "Run" to the beginning. Might make sense to always append this if you are trying to grep the files for some reason

**Comment by shairozan** [comment]
@dpastoor Have it in #142. Trying to find our gitflow document since I'm targeting develop with it from the issue branch. Just want to make sure I'm going in the right direction.

**Comment by shairozan** [comment]
Got this merged into develop. I think once we have approval for the exit code branch we should be good to cut a new release candidate.

**Comment by shairozan** [comment]
Available in https://github.com/metrumresearchgroup/babylon/releases/tag/v2.1.0-rc2

**Comment by shairozan** [comment]
Released in version 2.1.0
